src/components/features/ProductSearch/ProductSearch.module.scss

```diff
@@ -25,13 +25,66 @@
       z-index: 0;
     }
 
-    select {
-      -webkit-appearance: none;
-      border: 0;
-      background-color: transparent;
-      padding: 5px 30px 5px 35px;
-      position: relative;
-      z-index: 1;
+    ol {
+      list-style-type:none;
+      padding:10px;
+      margin:10px;
+      text-align:center;
+    }
+
+    ol a {
+      display:block;
+      text-decoration:none;
+      color:#000;
+      padding:0 5px;
+      font-size: 14px;
+      padding: 10px;
+    }
+
+    ol > li {
+      float:left;
```

Comment:
zamiast `float` może lepiej użyć position absolute i to ustawić względem relatywnego rodzica? To też powinno rozwiązać aktualny problem skakania całej sekcji.